CMakeLists.txt

```diff
@@ -81,9 +84,23 @@ add_definitions (-DBOOST_ASIO_DISABLE_BOOST_DATE_TIME=1)
 add_definitions (-DBOOST_ASIO_DISABLE_BOOST_REGEX=1)
 add_definitions (-DBOOST_COROUTINES_NO_DEPRECATION_WARNING=1)
 
-include_directories (${BOOST_ROOT})
+OPTION(FIND_BOOST "Build beast against an existing Boost install, rather than in-tree")
```

Comment:
Ugly but necessary workaround until boost officially supports cmake. In Boost.CMake, the projects just unashamedly use `find_package`, but the superproject disables the in-tree versions with [`bcm_ignore_package`](http://bcm.readthedocs.io/en/latest/src/BCMIgnorePackage.html#bcm-ignore-package).